File: sample/src/main/java/contacts/sample/cheatsheet/accounts/java/UpdateLocalRawContactsAccountsActivity.java
package contacts.sample.cheatsheet.accounts.java;

import android.accounts.Account;
import android.app.Activity;

import contacts.core.ContactsFactory;
import contacts.core.accounts.AccountsLocalRawContactsUpdate;
import contacts.core.entities.RawContact;

public class UpdateLocalRawContactsAccountsActivity extends Activity {

    AccountsLocalRawContactsUpdate.Result associateLocalRawContactToAccount(
            RawContact localRawContact, Account account
    ) {
        return ContactsFactory.create(this)
                .accounts()
                .updateLocalRawContactsAccount()
                .addToAccount(account)
                .localRawContacts(localRawContact)
                .commit();
    }
}

File: sample/src/main/java/contacts/sample/cheatsheet/accounts/kotlin/UpdateLocalRawContactsAccountsActivity.kt
package contacts.sample.cheatsheet.accounts.kotlin

import android.accounts.Account
import android.app.Activity
import contacts.core.Contacts
import contacts.core.accounts.AccountsLocalRawContactsUpdate
import contacts.core.entities.RawContact

class UpdateLocalRawContactsAccountsActivity : Activity() {

    fun associateLocalRawContactToAccount(
        localRawContact: RawContact, account: Account
    ): AccountsLocalRawContactsUpdate.Result = Contacts(this)
        .accounts()
        .updateLocalRawContactsAccount()
        .addToAccount(account)
        .localRawContacts(localRawContact)
        .commit()
}